src/main/java/com/minecolonies/coremod/colony/requestsystem/requesters/IBuildingBasedRequester.java

```diff
@@ -0,0 +1,14 @@
+package com.minecolonies.coremod.colony.requestsystem.requesters;
+
+import com.minecolonies.api.colony.requestsystem.manager.IRequestManager;
+import com.minecolonies.api.colony.requestsystem.request.IRequest;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.colony.requestsystem.request.IRequest'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)